src/components/CookieWarning.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash/fp'
+import { useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { ButtonPrimary, ButtonSecondary, Link } from 'src/components/common'
+import { Ajax } from 'src/libs/ajax'
+import { signOut } from 'src/libs/auth'
+import colors from 'src/libs/colors'
+import * as Nav from 'src/libs/nav'
+import { getLocalPref, setLocalPref } from 'src/libs/prefs'
+import { authStore } from 'src/libs/state'
+import * as Utils from 'src/libs/utils'
+
+
+export const cookiesAcceptedKey = 'cookiesAccepted'
+
+export const CookieWarning = () => {
```

Comment:
Usually we might have this as a default export, since the file and the component have the same name.

Also, we prefer blank lines between functions in the body, for readability. Basically, anything multiline at the top level of the component should be padded.

Reply:
The linter disagreed. It actually removed the blank lines I put between functions in the body, otherwise I would have done that.

Reply:
Nevermind, my linter was unhappy about spaces, fixed that.